source/frontend/src/features/mapSideBar/disposition/router/DispositionRouter.tsx

```diff
@@ -48,6 +51,18 @@ export const DispositionRouter: React.FC<IDispositionRouterProps> = props => {
         <Route path={`${stripTrailingSlash(path)}/property`}>
           <></>
         </Route>
+        <AppRoute
+          exact
+          path={`${stripTrailingSlash(path)}/${FileTabType.OFFERS_AND_SALE}/addOffer`}
```

Comment:
I'm pretty sure we are going to want something like:
${stripTrailingSlash(path)}/${FileTabType.OFFERS_AND_SALE}/offers/new
(new seems to be our standard naming)
why? because for edit we will want to include the specific id of the offer that is being edited, and we want our create/edit paths to follow a similar format, ie:
${stripTrailingSlash(path)}/${FileTabType.OFFERS_AND_SALE}/offers/1

Reply:
will fix this on my next PR.